bindinfo/bind_test.go

```diff
@@ -525,23 +526,6 @@ func (s *testSuite) TestCaptureBaselinesDefaultDB(c *C) {
 	c.Assert(len(tk.Se.GetSessionVars().StmtCtx.IndexNames), Equals, 0)
 }
 
-func (s *testSuite) TestUseMultiplyBindings(c *C) {
-	tk := testkit.NewTestKit(c, s.store)
-	s.cleanBindingEnv(tk)
-	tk.MustExec("use test")
-	tk.MustExec("drop table if exists t")
-	tk.MustExec("create table t(a int, b int, c int, index idx_a(a), index idx_b(b), index idx_c(c))")
-	tk.MustExec("insert into t values (1,1,1), (2,2,2), (3,3,3), (4,4,4), (5,5,5)")
-	tk.MustExec("analyze table t")
-	tk.MustExec("create binding for select * from t where a >= 1 and b >= 1 and c = 0 using select * from t use index(idx_a) where a >= 1 and b >= 1 and c = 0")
-	tk.MustExec("create binding for select * from t where a >= 1 and b >= 1 and c = 0 using select * from t use index(idx_b) where a >= 1 and b >= 1 and c = 0")
```

Comment:
We can keep it. To check the behavior when a sql bind is created multiple times.

Reply:
"the behavior when a sql bind is created multiple times" has been checked in following tests. This test check the plan result, I think it's meaningless.